PrivateCloud.DiagnosticInfo/PrivateCloud.DiagnosticInfo.psd1

```diff
@@ -10,7 +10,7 @@
 RootModule = 'PrivateCloud.DiagnosticInfo.psm1'
 
 # Version number of this module.
-ModuleVersion = '1.1.26'
+ModuleVersion = '1.1.27'

```

Comment:
Please update this with latest signed bits. (have updated the version to 1.1.28)